src/runtime_src/core/tools/common/reports/ReportAiePartitions.cpp

```diff
@@ -79,6 +114,7 @@ getPropertyTree20202(const xrt_core::device* _pDevice,
   boost::property_tree::ptree pt;
   pt.put("description", "AIE Partition Information");
   pt.add_child("partitions", populate_aie_partition(_pDevice));
+  pt.put("total_col_utilization", calculate_col_utilization(_pDevice, pt));
```

Comment:
Can you move this under description so it shows up before the partition info in the json?

Reply:
I agree that it would look better if it showed up beforehand; the main reason I put it after, however, is so that all the partition info is properly organized after the call to populate_aie_partition and I can cleanly use the separate calculate_col_utilization function. I think one possible alternative with minimal change could be to temporarily store the ptree result of populate_aie_partition before adding it to `pt` after calculating utilization? Please let me know what you think!